import_osm.sh

```diff
@@ -2,176 +2,177 @@
 
 cd `dirname $0`
 
-# vars
-DBHOST='127.0.0.1'
-DBNAME='pfb'
-OSMPREFIX='cambridge'
-OSMFILE='/vagrant/data/cambridge.osm'
+NB_POSTGRESQL_HOST="${NB_POSTGRESQL_HOST:-127.0.0.1}"
+NB_POSTGRESQL_DB="${NB_POSTGRESQL_DB:-pfb}"
+NB_POSTGRESQL_USER="${NB_POSTGRESQL_USER:-gis}"
+NB_POSTGRESQL_PASSWORD="${NB_POSTGRESQL_PASSWORD:-gis}"
+NB_OSMPREFIX="${NB_OSMPREFIX:-cambridge}"
+NB_OSMFILE="${NB_OSMFILE:-/vagrant/data/cambridge.osm}"
 
 # drop old tables
 echo 'Dropping old tables'
-psql -h $DBHOST -U gis -d ${DBNAME} \
-  -c "DROP TABLE IF EXISTS received.${OSMPREFIX}_ways;"
-psql -h $DBHOST -U gis -d ${DBNAME} \
-  -c "DROP TABLE IF EXISTS received.${OSMPREFIX}_ways_intersections;"
-psql -h $DBHOST -U gis -d ${DBNAME} \
-  -c "DROP TABLE IF EXISTS received.${OSMPREFIX}_relations_ways;"
-psql -h $DBHOST -U gis -d ${DBNAME} \
-  -c "DROP TABLE IF EXISTS received.${OSMPREFIX}_osm_nodes;"
-psql -h $DBHOST -U gis -d ${DBNAME} \
-  -c "DROP TABLE IF EXISTS received.${OSMPREFIX}_osm_relations;"
-psql -h $DBHOST -U gis -d ${DBNAME} \
-  -c "DROP TABLE IF EXISTS received.${OSMPREFIX}_osm_way_classes;"
-psql -h $DBHOST -U gis -d ${DBNAME} \
-  -c "DROP TABLE IF EXISTS received.${OSMPREFIX}_osm_way_tags;"
-psql -h $DBHOST -U gis -d ${DBNAME} \
-  -c "DROP TABLE IF EXISTS received.${OSMPREFIX}_osm_way_types;"
-psql -h $DBHOST -U gis -d ${DBNAME} \
-  -c "DROP TABLE IF EXISTS scratch.${OSMPREFIX}_cycwys_ways;"
-psql -h $DBHOST -U gis -d ${DBNAME} \
-  -c "DROP TABLE IF EXISTS scratch.${OSMPREFIX}_cycwys_ways_vertices_pgr;"
-psql -h $DBHOST -U gis -d ${DBNAME} \
-  -c "DROP TABLE IF EXISTS scratch.${OSMPREFIX}_cycwys_relations_ways;"
-psql -h $DBHOST -U gis -d ${DBNAME} \
-  -c "DROP TABLE IF EXISTS scratch.${OSMPREFIX}_cycwys_osm_nodes;"
-psql -h $DBHOST -U gis -d ${DBNAME} \
-  -c "DROP TABLE IF EXISTS scratch.${OSMPREFIX}_cycwys_osm_relations;"
-psql -h $DBHOST -U gis -d ${DBNAME} \
-  -c "DROP TABLE IF EXISTS scratch.${OSMPREFIX}_cycwys_osm_way_classes;"
-psql -h $DBHOST -U gis -d ${DBNAME} \
-  -c "DROP TABLE IF EXISTS scratch.${OSMPREFIX}_cycwys_osm_way_tags;"
-psql -h $DBHOST -U gis -d ${DBNAME} \
-  -c "DROP TABLE IF EXISTS scratch.${OSMPREFIX}_cycwys_osm_way_types;"
+psql -h $NB_POSTGRESQL_HOST -U ${NB_POSTGRESQL_USER} -d ${NB_POSTGRESQL_DB} \
```

Comment:
It might be helpful to assign a `PSQL_COMMAND` (or similar) variable to

`psql -h $NB_POSTGRESQL_HOST -U ${NB_POSTGRESQL_USER} -d ${NB_POSTGRESQL_DB}`

so that you can just call

`$PSQL_COMMAND  -c "DROP TABLE IF EXISTS received.${NB_OSMPREFIX}_ways;`

for example.